x-pack/test/security_solution_endpoint/page_objects/ingest_manager_create_datasource_page.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export function IngestManagerCreateDatasource({ getService }: FtrProviderContext) {
```

Comment:
Ingest Management Team:
I'm thinking that once you have Functional tests in place, that this PageObject module should live and be owned by the Ingest team. We would continue to consume it, but I think it makes more sense for it to be managed out of Ingest.

Reply:
gotcha